client/src/components/OverviewComponents/MainImage.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+
+class MainImage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
are you planning on using state in the class component? can leave for now if not and make note to reafactor later during tech debt time